core/snippets/snippets_insert.py

```diff
@@ -47,3 +48,23 @@ def insert_snippet_by_name_with_phrase(name: str, phrase: str):
             )
 
         actions.user.insert_snippet_by_name(name, substitutions)
+
+    def insert_wrapper_snippet(
+        body: str,
+        target: Any,
+        variable_name: str,
+        scope: str = None,
+    ):
+        """Wrap the target with snippet"""
+
+    def insert_wrapper_snippet_by_name(name: str, target: Any):
```

Comment:
Any objection to calling this `wrap_with_snippet_by_name`, and the above `wrap_with_snippet`? cc/ @knausj85

Reply:
I could go either way. @knausj85 any thoughts?

Reply:
I kinda-sorta prefer the more namespace-y names as-is

Reply:
Action names updated
@pokey better?